src/DataTypes/Either/Either.cs

```diff
@@ -62,5 +62,13 @@ private Either(L left)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool operator !(Either<L, R> value)
             => value.IsLeft;
+
+        [Pure]
+        public R UnwrapRight()

```

Comment:
I don't want to bore you, but what do you think about something like map,bind,unwrap and maleft,bindleft,unwrapleft just for summery 😁